documentation/src/main/asciidoc/topics/proc_download_hotrod_java_client.adoc

```diff
@@ -1,6 +1,6 @@
 [id='download_java_client']
-= Downloading the Hot Rod Java Client
-{brandname} provides a reference Hot Rod Java client implementation.
+= Getting the Hot Rod Java Client
+{brandname} provides a Hot Rod Java client implementation.
 
 //Community
 ifndef::productized[]
```

Comment:
we no longer zip the java hot rod client. Users must use the maven coordinates.

Reply:
thanks @pruivo fixed